src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -60,9 +61,9 @@ static DefaultServiceControl()
             cache = new MemoryCache("ServiceControlReponses", new NameValueCollection(1) { { "cacheMemoryLimitMegabytes", settings.CacheSize.ToString() } });
         }
 
-        public bool IsAlive()
+        public async Task<bool> IsAlive()
         {
-            var version = GetVersion();
+            var version = await GetVersion().ConfigureAwait(false);
```

Comment:
We only call `ConfigureAwait(false)` here, because this code has no knowledge of the UI and will therefore never use the UI thread.